plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout.spec.js

```diff
@@ -175,10 +175,29 @@ test.describe(
 			await expect( page.locator( '#payment_method_cod' ) ).toBeEnabled();
 		} );
 
+		await test.step( 'Complete the checkout form', async () => {
```

Comment:
Looks like E2E tests are failing because of the `await` here:

https://github.com/woocommerce/woocommerce/actions/runs/10373657102/job/28719326875?pr=49696

Reply:
@dkotter hmmm .. I had all of these errors cleared up and looks like when `trunk` was merged in that we have a few more .. I will look into them and straighten them out.